src/main/java/TasksJavaCore3/Task1.java

```diff
@@ -0,0 +1,44 @@
+package TasksJavaCore3;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Task1 {
+
+    //Task 1.1
+    public static void printStreamNumbers(){
+        Stream<Integer> stream = Stream.of(1,2,3,4,5,6,7,8,9);
+        stream.forEach(System.out::println);
+    }
+
+    //Task 1.2
+    public static List<Integer> generateRandomNumbers(){
+        Stream<Integer> randomNumbers = Stream.generate(() -> new Random().nextInt(100))
```

Comment:
instead of creating a new Random object for each number, initialize one Random instance outside the stream generation